include/ensmallen_bits/callbacks/progress_bar.hpp

```diff
@@ -216,9 +216,8 @@ class ProgressBar
         output << ".";
       }
     }
-
     const size_t stepTime = epochTimer.toc() / (double) epochSize * 1000;
-    output << "] " << progress << "% - " << (size_t) epochTimer.toc() % 60
+    output << "] " << progress << "% - " << epochTimer.toc()
         << "s " << stepTime << "ms/step " << "- loss: " << objective  <<  "\n";
```

Comment:
Suggest to demarcate the various fields explicitly, so they appear more clearly to users. For example, change:  
`<< "s " << stepTime << "ms/step " << "- loss: " << objective  <<  "\n";`
to:  
`<< "s;  " << stepTime << "ms/step;  " << "- loss: " << objective  <<  "\n";`

Reply:
Good idea, I'll adapt the format.